algorithms/src/sorting/Kokkos_SortPublicAPI.hpp

```diff
@@ -217,8 +274,14 @@ std::enable_if_t<Kokkos::is_execution_space<ExecutionSpace>::value> sort(
   bin_sort.sort(exec, view, begin, end);
 }
 
+// clang-format off
 template <class ViewType>
-void sort(ViewType view, size_t const begin, size_t const end) {
+void sort(ViewType view,
+	  size_t const begin,
+	  size_t const end)
+{
+  // clang-format on
+
```

Comment:
I suppose your intent is to defer to `sort(ExecutionSpace, ViewType, size_t, size_t)` do check `ViewType`.
This is not consistent with what we do for `sort(ViewType)`.

Reply:
i originally had it here too, then removed it. i will add it back.